libr/core/cbin.c

```diff
@@ -1867,13 +1873,17 @@ static int bin_symbols_internal(RCore *r, int mode, ut64 laddr, int va, ut64 at,
 		if (IS_MODE_RAD (mode)) {
 			r_cons_printf ("fs exports\n");
 		} else if (IS_MODE_NORMAL (mode)) {
-			r_cons_printf (printHere ? "" : "[Exports]\n");
+			r_cons_printf (printHere ? "" : "[Exports]\n"
+			                                "Num Paddr      Vaddr      Bind"
+																			"     Type Size Name\n");
 		}
 	} else if (!at && !exponly) {
 		if (IS_MODE_RAD (mode)) {
 			r_cons_printf ("fs symbols\n");
 		} else if (IS_MODE_NORMAL (mode)) {
-			r_cons_printf (printHere ? "" : "[Symbols]\n");
+			r_cons_printf (printHere ? "" : "[Symbols]\n"
+			                                "Num Paddr      Vaddr      Bind"
+																			"     Type Size Name\n");
```

Comment:
?

please just use one line, it makes everything much more readable (in this case there's also the huge indentation that doesn't help)

Reply:
Had been trying to conform to the 78 char limit in the style guide. Will change this. (And I think the indent was my editor, will fix that too)

Reply:
Yeah, no problem. In general the code shouldn't be too wide (78 chars at max), but if you have to split strings such as in this case, it's better to have one longer string than splitting it.